packages/contracts/contracts/Unirep.sol

```diff
@@ -63,6 +63,8 @@ contract Unirep is IUnirep, zkSNARKHelper, VerifySignature {
     uint256 public nextAttesterId = 1;
     // Mapping of the airdrop amount of an attester
     mapping(address => uint256) public airdropAmount;
+    // Mapping of existing nullifiers to the epoch of emitted
+    mapping(uint256 => uint256) public nullifierExists;
```

Comment:
What do you think of naming this `spentNullifiers`?

Reply:
Or `usedNullifiers`?
Maybe spent sounds more like spending reputation nullifier.

Reply:
Sure, `nullifierExists` just sounds more like a function than a variable.